### DISCUSSION TITLE
新增布局保存建议

### DISCUSSION THREAD

**Discussion by wdubaiyu** [eiyooooo/Easycontrol_For_Car#17] [Ideas]
每次开车都要重新连接，连接后还要重新布局导航和音乐位置。地图我安装的车机版，大小还需要重新调整。如果能增加布局，连接后恢复上一次布局就好了。

**Comment by eiyooooo** [comment]
现在会自动保存最后的位置的，多个位置保存为配置文件功能正在规划中

**Reply by wdubaiyu** [reply]
易控软件打开后，能自动隐藏车机导航栏吗。比亚迪下面那个导航栏太占位置了，挤压了手机镜像/流转的app纵向空间。

**Reply by eiyooooo** [reply]
> 易控软件打开后，能自动隐藏车机导航栏吗。比亚迪下面那个导航栏太占位置了，挤压了手机镜像/流转的app纵向空间。

悬浮窗模式无法自动隐藏车机导航栏

全屏模式开启沉浸式全屏模式后软件可以请求系统隐藏（可能受限于车机系统限制无法成功）
